src/Http/Controllers/VoyagerMediaController.php

```diff
@@ -255,8 +255,8 @@ public function upload(Request $request)
                 'image/svg+xml',
             ];
             if (in_array($request->file->getMimeType(), $imageMimeTypes)) {
-                $image = Image::make($realPath.$file);
-
```

Comment:
Can you restore the blank lines as the original?
It makes diff much more readable, you can compare with this: https://github.com/MrCrayon/voyager/commit/1a55e5653bfb3d09ce99ef812070bb35fe616d11